api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -715,37 +715,8 @@ public static int compareWithNullAsLatest(Date d1, Date d2) {
 	}
 	
 	public static Set<Concept> conceptSetHelper(String descriptor) {
-		Set<Concept> ret = new HashSet<>();
-		if (descriptor == null || descriptor.length() == 0) {
-			return ret;
-		}
-		ConceptService cs = Context.getConceptService();
-		
-		for (StringTokenizer st = new StringTokenizer(descriptor, "|"); st.hasMoreTokens();) {
-			String s = st.nextToken().trim();
-			boolean isSet = s.startsWith("set:");
-			if (isSet) {
-				s = s.substring(4).trim();
-			}
-			Concept c = null;
-			if (s.startsWith("name:")) {
-				String name = s.substring(5).trim();
-				c = cs.getConceptByName(name);
-			} else {
-				try {
-					c = cs.getConcept(Integer.valueOf(s.trim()));
-				}
-				catch (Exception ex) {}
-			}
-			if (c != null) {
-				if (isSet) {
-					List<Concept> inSet = cs.getConceptsByConceptSet(c);
-					ret.addAll(inSet);
-				} else {
-					ret.add(c);
-				}
-			}
-		}
+		Set<Concept> ret = new HashSet<>(conceptListHelper(descriptor));
```

Comment:
Why did you remove the above code for this method and instead delegated `conceptListHelper(arg..)`; do you mean these two methods treat these collections in the same way?